stories/Molecules/FooterNavigation/FooterIcons/FooterIcons.jsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import './footerIcons.scss';
 
+export const variant_options = {
+  default: '',
+  inverted: 'inverted',
+};
+
+const cls = (...classes) => classes.filter(Boolean).join(' ');
 export const FooterIcons = ({ ...args }) => (
-  <ul className={['footer__icons', `${args.variant === 'black' ? '' : args.variant}`].join(' ')}>
+  <ul className={cls('footer__icons', `${variant_options[`${args.variant}`]}`)}>
     <li><a href="#" className="facebook">facebook</a></li>
-    <li><a href="#" className="linkedin">linkedIn</a></li>
-    <li><a href="#" className="instragram">instragram</a></li>
+    <li><a href="#" className="undp-linkedin">linkedIn</a></li>
+    <li><a href="#" className="instagram">instragram</a></li>
```

Comment:
@Shubham-Sharma-77 Any specific reason why this needs to be `undp-linkedin`? Otherwise change this to just `linkedin`. 
Also change spelling to `instagram`. 
For these `a` tags add a `title` attribute specifying the name of the social icon, eg. "facebook", "twitter", etc.